.appveyor.yml

```diff
@@ -38,3 +38,25 @@ init:
 
 build_script:
   - ctest -VV -S cmake/ctest/script_ci.ctest
+
+# scripts to run after build
+after_build:
+  - 7z a openrw-%appveyor_repo_branch%.zip %APPVEYOR_BUILD_FOLDER%\build\rwgame\Release\
```

Comment:
Consider adding ```*``` in the end of the path to get rid of unnecessary folders in archive